packages/amplify-provider-awscloudformation/src/utility-functions.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
     for (const resource of options.resourcesToBuild) {
       await transformResourceWithOverrides(context, resource);
     }
+    await transformResourceWithOverrides(context);
```

Comment:
why the repeat call? It's being called above too in the loop.

Reply:
reapeat call is for root stack when no resources are provided

Reply:
if resource is passed , then it will build only that resource
if not , then it will build root stack